modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,29 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.1 or WebP returns 82,
+ * Otherwise returns default quality.
+ *
+ * @since n.e.x.t
+ *
+ * @global string $wp_version The WordPress version string.
+ *
+ * @param int    $quality   Quality level between 1 (low) and 100 (high).
+ * @param string $mime_type Image mime type.
+ * @return int The updated quality for mime types.
+ */
+function webp_uploads_set_wp_image_editor_quality( $quality, $mime_type ) {
```

Comment:
A bit more precise name: This function is focused on changing WebP image quality, not any image quality.
```suggestion
function webp_uploads_modify_webp_quality( $quality, $mime_type ) {
```